src/lib.rs

```diff
@@ -1,40 +1,55 @@
 // Copyright 2019 Intel Corporation. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 or BSD-3-Clause
 
+//! `vmm-sys-util` is a collection of utilites to make vmm implementation
+//! more convenient.
+
+#![deny(missing_docs)]
+
 extern crate libc;
 
 mod tempdir;
 
 #[macro_use]
-pub mod ioctl;
+mod ioctl;
```

Comment:
I prefer having the functionality exported with modules because in this case the modules actually have the purpose of doing a logical separation of components. I know that implies documenting the modules as well, but it shouldn't be super hard to do it.

Reply:
Agreed. Public modules would be good than directly publishing APIs.